includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -537,13 +551,15 @@ public function prepare_validation_error( array $error = [], array $data = [] )
 
 		if ( $node instanceof DOMElement ) {
 			if ( ! isset( $error['code'] ) ) {
-				$error['code'] = AMP_Validation_Error_Taxonomy::INVALID_ELEMENT_CODE;
+				$error['code'] = AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_TAG;
```

Comment:
Good point. On one hand, I think this should just be removed entirely, and let the `code` be `UNKNOWN_ERROR`. Each sanitizer should be responsible for indicating the code, specifically.

Otherwise, I think this should be scoped with what you discussed in https://github.com/ampproject/amp-wp/pull/3780#discussion_r350821157. It's bound up with generating PHP objects from the spec.

So I think we should do it, but probably not in the scope of this PR.